doc/EFFECTS_JSON.md

```diff
@@ -212,9 +212,16 @@ Because the "dur_add_perc" = 150 in the example above, the second addition adds
 a total value of 250 ticks from the two. This can also be below 100%, and should be able to even be negative, leading to
 future applications decreasing the overall time left.
 
+### Duration control
```

Comment:
I don't follow what you're trying to do here, if you increase the duration every turn, the effect is permanent, and that's all you seem to be doing with it.  Also it isn't implemented, it just sets a value on the effect and never uses it.